site/en/Test.md

```diff
@@ -0,0 +1,41 @@
+
+All pilot libraries have some know-how and experience with OCR, as they have already produced full texts at least at project level or via service providers. The extent to which OCR, which is regarded as important, should be carried out independently in future and firmly anchored in the digitisation workflow, is currently discussed in the libraries. Concerning the target groups of the full texts there are different opinions in the testing institutions. While one third generally name humanities scholars, another third would like to serve a very broad target group (humanities, digital humanities, computer linguistics and economics). The remaining libraries only see relatively few, specialised users (Digital Humanities or Computational Linguistics) as the target group for OCR texts.
+
+The pilot libraries demand the following features of an OCR software:
+* High recognition rate of layout and text
+* Cost-effective use
+* Quick adaptability/troubleshooting
+* Modularity
+* Output in standard formats
+* Connection to existing workflows
+* Well documented interfaces
+* Word coordinates
+* Trainability
+* Extensive GT corpora
+
+High quality of text recognition is most important, whereas the other characteristics are only named each by some of the pilot libraries and should be considered as desired but subordinate optional features.
+
+
+## Evaluation of the software tests
+In order to ensure the comparability of the individual tests in the pilot libraries, a questionnaire distributed to the pilot libraries at the beginning of the test. In this questionnaire, the general conditions of the test run, e.g. the technical equipment used and the tested OCR-D processors, as well as the documentation of the software, interfaces, functionality and usability of the software, its possibilities for being integrated into existing workflows and the required output formats are recorded. Furthermore, recognition quality, functionality and usability, open requirements and positive features of the OCR-D software as well as the results of the test were to be described.
+
+During the test phase, the various options for installing the OCR-D software with and without a docker container were tried out and the software was successfully installed on a wide range of differently powerful, partly virtual servers. For non-Intel computers (ARM, PowerPC64) this was more complicated and time consuming, as individual Python packages were not executable on these computers and had to be adapted manually. The complete installation of all available OCR-D processors (``ocrd_all``), which was developed during the test phase, was confirmed as the simplest and least complex variant and was therefore the most recommendable. No pilot library integrated the OCR-D software In a workflow software such as Kitodo, as this would have meant too much work for a simple test run. 
```

Comment:
docker -> Docker

differently powerful -> servers with varying processing power, some of them virtual.

Non-Intel computers -> Non-Intel CPU architectures

were not executable  on -> were not packaged for

One backtick is enough for inline monospace.

"simplest and least complex" seems redundant.